src/main/java/com/yahoo/validatar/report/email/EmailFormatter.java

```diff
@@ -0,0 +1,247 @@
+        {
+            accepts(RECIPIENT_EMAILS, "Comma-separated list of emails to send reports")
+                    .withRequiredArg()
+                    .describedAs("Report recipients' emails");
+            accepts(SENDER_NAME, "Name of sender displayed to report recipients")
+                    .withRequiredArg()
+                    .defaultsTo("Validatar");
+            accepts(FROM_EMAIL, "Email shown to recipients as 'from'")
+                    .withRequiredArg();
+            accepts(REPLY_TO, "Email to which replies will be sent")
+                    .withRequiredArg();
+            accepts(SMTP_HOST, "Email SMTP host name")
+                    .withRequiredArg();
+            accepts(SMTP_PORT, "Email SMTP port")
+                    .withRequiredArg();
+            allowsUnrecognizedOptions();
+        }
+    };
+
+    /**
+     * List of recipient emails for Validatar reports.
+     */
+    private List<String> recipientEmails;
+    /**
+     * Sender name to display to report recipients.
+     */
+    private String senderName;
+    /**
+     * From email to display to report recipients.
+     */
+    private String fromEmail;
+    /**
+     * Email to which report email replies are sent.
+     */
+    private String replyTo;
+    /**
+     * SMTP server host.
+     */
+    private String smtpHost;
+    /**
+     * SMTP server port.
+     */
+    private int smtpPort;
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public boolean setup(String[] arguments) {
+        OptionSet options = PARSER.parse(arguments);
+        senderName = (String) options.valueOf(SENDER_NAME);
+        fromEmail = (String) options.valueOf(FROM_EMAIL);
+        replyTo = (String) options.valueOf(REPLY_TO);
+        smtpHost = (String) options.valueOf(SMTP_HOST);
+        smtpPort = Integer.parseInt((String) options.valueOf(SMTP_PORT));
+        recipientEmails = (List<String>) options.valuesOf(RECIPIENT_EMAILS);
+        return true;
+    }
+
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Render the report HTML using Jtwig and send the result
+     * to the recipient emails.
+     */
+    @Override
+    public void writeReport(List<TestSuite> testSuites) throws IOException {
+        if (testSuites == null) {
+            testSuites = Collections.emptyList();
+        }
+        log.info("Sending report email for {} test suites", testSuites.size());
+        List<TestSuiteModel> testList = new ArrayList<>(testSuites.size());
+        boolean hasError = false;
+        for (TestSuite testSuite : testSuites) {
+            TestSuiteModel testSuiteModel = new TestSuiteModel(testSuite);
+            hasError = hasError || !testSuiteModel.allPassed();
+            testList.add(testSuiteModel);
+        }
+        JtwigTemplate template = JtwigTemplate.classpathTemplate("templates/email.twig");
+        JtwigModel model = JtwigModel.newModel()
+                .with("error", hasError)
```

Comment:
Nitpick: The convention for the project is to line up the newlines with the line up (so ```.with``` lines up with ```.newModel``` and so on). I don't agree with that convention anymore since that alignment forces you to reindent everytime the length of the first line changes but I'd like to keep it all consistent if possible.